mmv1/products/vertexai/Schedule.yaml

```diff
@@ -0,0 +1,323 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: Schedule
+min_version: beta
+base_url: projects/{{project}}/locations/{{location}}/schedules
+create_url: projects/{{project}}/locations/{{location}}/schedules
+self_link: 'projects/{{project}}/locations/{{location}}/schedules/{{name}}'
+update_verb: :PATCH
```

Comment:
there is no "update" available for vertex AI schedules. However there is pause/resume. How can this be represented?

Reply:
Is there a particular URL that is used for pause/resume? Otherwise do you have a field that controls pause/resume?

Reply:
Yes there are separate URLs for pause and resume:

`POST https://{service-endpoint}/v1beta1/{name}:pause` (see [docs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.schedules/pause)
`POST https://{service-endpoint}/v1beta1/{name}:resume` (see [docs](https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.schedules/resume))

Reply:
I haven't seen a situation like this, I suspect some custom code needs to be added. Refer to https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform

Reply:
I have had a go at copying the approach from the Cloud Scheduler resource